lxd/device/nic_bridged.go

```diff
@@ -681,8 +681,10 @@ func (d *nicBridged) setFilters() (err error) {
 
 	// If parent bridge is unmanaged check that IP filtering isn't enabled.
 	if err == db.ErrNoSuchObject || n == nil {
-		if shared.IsTrue(d.config["security.ipv4_filtering"]) || shared.IsTrue(d.config["security.ipv6_filtering"]) {
-			return fmt.Errorf("IP filtering requires using a managed parent bridge")
+		if shared.IsTrue(d.config["security.ipv4_filtering"]) && shared.IsBlank(d.config["ipv4.address"]) ||
+			(shared.IsTrue(d.config["security.ipv6_filtering"]) && shared.IsBlank(d.config["ipv6.address"])) {
+			return fmt.Errorf("IP filtering requires using a managed parent bridge." +
```

Comment:
We usually just let error messages like this continue on a single line without the need for concatenation as it makes it easier to grep for them.